cmd/wsm/wsm.go

```diff
@@ -190,6 +190,17 @@ type LocalSpec struct {
 	Values values.Values `json:"values" yaml:"values"`
 }
 
+func getChart(airgapped bool, chartPath string, _spec *spec.Spec) spec.Chart {
+	if airgapped {
+		return spec.Chart{Path: chartPath}
+	}
+	return spec.Chart{
+		URL:     _spec.Chart.URL,
```

Comment:
why `_` haven't seen this style before

Reply:
oh i see name conflict, I won't recommend doing this, maybe call it s or something